_docs/geyser/setup.md

```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
 
-If you're still having problems with Geyser not working or giving you an "Unable to connect to world" error, see the [Common Issues](/geyser/common-issues/) page.\
-For more information, take a look at the [Understanding the Config](/geyser/understanding-the-config/) page, and the [FAQ](/geyser/faq/) page.\
-And if you still have questions, feel free to join the [Discord](https://discord.geysermc.org) if you haven't already.
 
 ## Prerequisites
-
 - The server you are connecting to has to support the latest version of Minecraft Java Edition (at this time this is Minecraft {{ site.data.versions.java }})
-The server itself does not have to be the latest version but does have to allow connections. If you're running the server on an older version, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/), but do note that only **1.12.2 and any version above is officially supported**.
+The server itself does not have to be the latest version but does have to allow connections. If you're running the server on an older version, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/), but do note that only **1.13.2 and any version above is officially supported**. For older versions, use Geyser Standalone, or Geyser on a proxy like Velocity or Bungeecord.
 - The device running Geyser must support Java 16 or later. If you need help installing or updating, please see [PaperMC's Java update page](https://paper.readthedocs.io/en/latest/java-update/index.html). If you're running a version of Paper that does not support Java 16 or later, you can add the flag `-DPaper.ignoreJavaVersion=true` to your startup Java arguments to allow Paper to run on Java 16. You can run Geyser standalone on another device if a server software cannot be updated to use Java 16.
 - If you are connecting to an online mode Java server, a paid Java account is required. If you are running the server, you can bypass this requirement for your server with [Floodgate](/floodgate/).
 - Your Bedrock client has to be a supported version - at this time that is Bedrock version(s) {{ site.data.versions.bedrock }}.
 - If you are running the server, you need to have a UDP port opened. See below for more instructions.
+- Currently, Geyser does not support key signing. To disable it (on servers running version 1.19 and higher), follow these instructions: Spigot, Paper, & all forks: Set `enfore-secure-profile: false` in the `server.properties` file. On BungeeCord: Set `enforce_secure_profile: false` in your `config.yml`. Using Velocity: set `force-key-authentication = false` in your `velocity.toml`. This will disable chat signing, which Geyser currently does not support. Be careful though: This will allow java players who bypass chat signing with mods such as NoChatReports to join the server, and make them unreportable. For more info on chat reporting, see this [article](https://gist.github.com/kennytv/ed783dd244ca0321bbd882c347892874).
```

Comment:
```suggestion
- Currently, Geyser does not support key signing. To disable it (on servers running version 1.19 and higher), follow these instructions: Spigot, Paper, & all forks: Set `enfore-secure-profile: false` in the `server.properties` file. On BungeeCord: Set `enforce_secure_profile: false` in your `config.yml`. Using Velocity: set `force-key-authentication = false` in your `velocity.toml`. This will disable chat signing, which Geyser currently does not support. Be careful though: This will allow Java players who bypass chat signing with mods such as NoChatReports to join the server, and make them unreportable. For more info on chat reporting, see this [article](https://gist.github.com/kennytv/ed783dd244ca0321bbd882c347892874).
```
caps